lib/rules/disallow-space-after-object-keys.js

```diff
@@ -30,18 +64,40 @@ module.exports = function() {};
 
 module.exports.prototype = {
 
-    configure: function(disallow) {
-        var modes = {
-            'ignoreSingleLine': 'ignoreSingleLine',
-            'ignoreMultiLine': 'ignoreMultiLine'
-        };
+    configure: function(options) {
+        if (typeof options !== 'object') {
+            assert(
+                options === true,
+                this.getOptionName() +
+                ' option requires a true value or an object'
+            );
+
+            var _options = {
+                allExcept: []
+            };
+
+            return this.configure(_options);
+        } else {
+            assert(
+                Array.isArray(options.allExcept),
+                this.getOptionName() +
+                ' option object requires allExcept array property'
+            );
+        }
+
+        this._exceptSingleline = options.allExcept.indexOf('singleline') > -1;
+        this._exceptMultiline = options.allExcept.indexOf('multiline') > -1;
+        this._exceptAligned = options.allExcept.indexOf('aligned') > -1;
+        assert(
+            !this._exceptMultiline || !this._exceptAligned,
+            this.getOptionName() +
+            ' option allExcept property cannot contain "aligned" and "multiline" at the same time'
+        );
         assert(
-            disallow === true || typeof disallow === 'string' && modes[disallow],
+            !this._exceptMultiline || !this._exceptSingleline,
             this.getOptionName() +
-            ' option requires true value requires one of the following values: ' +
-            Object.keys(modes).join(', ')
+            ' option allExcept property cannot contain "singleline" and "multiline" at the same time'
```

Comment:
For some reason, we use "\`" symbol for those errors, like that "\`singleline\`", i have no idea why though